app/assets/stylesheets/components/syntax.scss

```
@@ -12,7 +12,8 @@
 }
 
 pre:not(.highlight),
-div.highlight {
+div.highlight,
+li pre.highlight {
```

Could you also add `blockquote` here as well, so the following example would be supported too:
```suggestion
li pre.highlight,
blockquote pre.highlight {
```
> Here's a blockquote with code block below:
> ```yml
> - name: Get a dad-joke
> ```